model_templates/cog.yaml

```diff
@@ -22,18 +22,20 @@ build:
     - "py7zr"
     - "fire"
     - "datasets"
-  run: 
+    - "git+https://github.com/huggingface/transformers.git@d2295708a64feff485bb40caf0decf7bae5a05a2"
+    - "peft==0.4.0"
+    - "tensorizer==1.0.1"
+    - "accelerate"
+    - "scipy"
+    - "bitsandbytes" # force upgrade afterwards?
+    - "https://r2.drysys.workers.dev/torch/torch-2.1.0a0+git3af011b-cp311-cp311-linux_x86_64.whl"
+    - "trl==0.5.0"
+  run:
     - curl -o /usr/local/bin/pget -L "https://github.com/replicate/pget/releases/download/v0.0.1/pget" && chmod +x /usr/local/bin/pget
-    - pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu118
-    - pip install git+https://github.com/huggingface/transformers.git@d2295708a64feff485bb40caf0decf7bae5a05a2
-    - pip install peft==0.4.0
-    - pip install tensorizer==1.0.1
-    - pip install accelerate
-    - pip install scipy
-    - pip install -q -U bitsandbytes
-    - pip install trl==0.5.0
-
+    # - pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu118
```

Comment:
at some point, I had read something somewhere (not sure where), that said exllama required a pytorch nightly build -- did that end up not being the case?

Reply:
yes, it does. `https://r2.drysys.workers.dev/torch/torch-2.1.0a0+git3af011b-cp311-cp311-linux_x86_64.whl` is a nightly build. unfortunately I don't know how to rename wheels in a way that makes it clear they're nightly or compiled in a special way while still being a valid wheel name